.github/workflows/release-publish.yml

```
@@ -8,8 +8,35 @@ on:
       - 'release/*'
 
 jobs:
+  build-amd64:
+    runs-on: blacksmith-4vcpu-ubuntu-2204
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup-nodejs-blacksmith
+
+      - name: Cache to GH
+        uses: actions/cache/save@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
+        with:
+          path: ./packages/**/dist
+          key: ${{ github.sha }}-release:build
+  build-arm64:
```

nit:
```suggestion
          key: ${{ github.sha }}-release:build

  build-arm64:
```

Do we have a linter for GH workflow files that would catch things like this? If not, something to consider in the future